Update n8n org README.md (#5)
n8n-io/.github: profile/README.md

### Hi there, this is n8n 👋

n8n is the company behind n8n, the product. We're building a workflow automation platform that gives technical teams the flexibility of code with the speed of no-code. With 400+ integrations, native AI capabilities, and a <a href="https://faircode.io/">fair-code</a> license, n8n lets you build powerful automations while maintaining full control over your data and deployments.

> - Check out <a href="https://github.com/n8n-io/n8n">our main project here.</a>

Reviewer: ```suggestion
- Check out [our main project here](https://github.com/n8n-io/n8n).
```

> - Interested in working for n8n? See our<a href="https://n8n.io/careers"> open positions</a>.

Reviewer: ```suggestion
- Interested in working at n8n? See our [open positions](https://n8n.io/careers).
```

Small thing, but "at" sounds a bit more welcoming than "for" to me.

> - Learn more how to <a href="https://docs.n8n.io/hosting/">self-host n8n in our docs</a>. 

Reviewer: ```suggestion
- Learn more how to [self-host n8n in our docs](https://docs.n8n.io/hosting/). 
```

> - Not interested in hosting n8n yourself? We also offer <a href="https://n8n.io/cloud/?utm_medium=referral&utm_source=github.com&utm_campaign=readme">n8n cloud</a> for a monthly fee. 

Reviewer: ```suggestion
- Not interested in hosting n8n yourself? We also offer [n8n cloud](https://n8n.io/cloud/?utm_medium=referral&utm_source=github.com&utm_campaign=readme) for a monthly fee. 
```

> - Both our team and our users have been building a <a href="https://n8n.io/workflows/?utm_medium=referral&utm_source=github.com&utm_campaign=readme">workflow template library</a> to help others easily get started with n8n automations. 

Reviewer: ```suggestion
- Both our team and our users have been building a [workflow template library](https://n8n.io/workflows/?utm_medium=referral&utm_source=github.com&utm_campaign=readme) to help others easily get started with n8n automation. 
```

![n8n.io - Screenshot](https://raw.githubusercontent.com/n8n-io/n8n/master/assets/n8n-screenshot-readme.png)

 We launched in 2019 with a ton of interest, and things have been going great since then:

- 📈 With 100m+ Docker pulls and a huge active user base/community - we have global traction

> - ⭐️ With over ~70k~ 80k stars, we are now part of the top 150 projects on GitHub

Reviewer: It might make sense to for us to  link to the main n8n repo here since this info is right there for users to verify. If we can make a workflow to automatically update this and the Docker pulls information as numbers climb, even better.

Author: I'll leave this for later 😇

- 🌱 We were Sequoia's first seed investment in Germany, and recently raised a $55m Series B round, led by Highland Capital

> We're on a mission to give technical people the powers of a 10x developer. <a href="https://n8n.io/careers">Join us!</a>

Reviewer: ```suggestion
We're on a mission to give technical people the powers of a 10x developer. [Join us!](https://n8n.io/careers)
```
